src/sysdep.h

```diff
@@ -87,7 +87,9 @@
 #include <memory.h>
```

Comment:
Just above this, this code resides:
```
#if     (defined (STDC_HEADERS) || defined (__GNU_LIBRARY__))
#include <stdlib.h>
```
That you had to comment out the extern char define below means you hit the wrong branch in that if.
So, why aren't you picking up the standard c headers in the configure script, I wonder?

Reply:
Well, the [AC_HEADER_STDC man page](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Particular-Headers.html#index-AC_005fHEADER_005fSTDC-621) makes a good case for ignoring the STDC_HEADERS completely and always including the stdlib.h:

> This macro is obsolescent, as current systems have conforming header files. New programs need not use this macro.

Sometimes we get carried away keeping the mud able to run on esoteric, non-standardscompliant operating systems. 

Instead of all those changes in this PR could you try out removing that `#if` along with the corresponding `#else` block and see if that works for you?

Reply:
Ok, I will try that.